src/dune_lang/dep_conf.ml

```diff
@@ -103,8 +105,13 @@ let decode files =
         , let+ () = Syntax.since Stanza.syntax (3, 1)
           and+ filename = filename in
           Include filename )
+      ; ( "order_only"
+        , let+ () = Syntax.since Stanza.syntax (3, 18)
```

Comment:
The last released version of Dune is 3.16; isn't 3.17 enough?
```suggestion
        , let+ () = Syntax.since Stanza.syntax (3, 17)
```

Reply:
If that is the case, then yes. I thought that 3.17 was released because the version was bumped in the dune codebase.